woohoo.rb

```diff
@@ -0,0 +1,12 @@
+[42, 999].inject(&:+) # should complain
+[42, 999].reduce(&:+) # should not complain
+
+[123, 42, 999].inject(&:+) # should complain
+[123, 42, 999].reduce(&:+) # should not complain
+
+[9].inject(&:+) # should complain
+[9].reduce(&:+) # should not complain
+[:aaaa].find { |_| true } # should not complain
+[:aaaa].detect { |_| true } # should complain
+
+"HOUND BITE ME!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.